static/app/views/dashboardsV2/widget/metricWidget/index.tsx

```diff
@@ -202,91 +217,135 @@ class MetricWidget extends AsyncView<Props, State> {
     }
 
     return (
-      <GlobalSelectionHeader
-        onUpdateProjects={this.handleProjectChange}
-        disableMultipleProjectSelection
-        skipLoadLastUsed
-      >
-        <StyledPageContent>
-          <Header
-            orgSlug={orgSlug}
-            title={title}
-            onChangeTitle={newTitle => this.handleFieldChange('title', newTitle)}
-          />
-          <Layout.Body>
-            <BuildSteps>
-              <BuildStep
-                title={t('Choose your visualization')}
-                description={t(
-                  'This is a preview of how your widget will appear in the dashboard.'
-                )}
-              >
-                <VisualizationWrapper>
-                  <StyledSelectField
-                    name="displayType"
-                    choices={[
-                      DisplayType.LINE,
-                      DisplayType.BAR,
-                      DisplayType.AREA,
-                    ].map(value => [value, displayTypes[value]])}
-                    value={displayType}
-                    onChange={value => {
-                      this.handleFieldChange('displayType', value);
-                    }}
-                    inline={false}
-                    flexibleControlStateSize
-                    stacked
-                  />
-                  <Card
-                    router={router}
-                    location={location}
-                    selection={selection}
-                    organization={organization}
-                    api={this.api}
-                    project={project}
-                    widget={{
-                      title,
-                      searchQuery,
-                      displayType,
-                      groupings: queries,
-                    }}
-                  />
-                </VisualizationWrapper>
-              </BuildStep>
-              <ChooseDataSetStep value={DataSet.METRICS} onChange={onChangeDataSet} />
-              <BuildStep
-                title={t('Begin your search')}
-                description={t('Select a query to compare projects, tags, etc.')}
-              >
-                <SearchQueryField
-                  api={this.api}
-                  tags={metricTags}
-                  orgSlug={orgSlug}
-                  projectSlug={project.slug}
-                  query={searchQuery}
-                  onSearch={newQuery => this.handleFieldChange('searchQuery', newQuery)}
-                  onBlur={newQuery => this.handleFieldChange('searchQuery', newQuery)}
+      <StyledPageContent>
+        <Header
+          orgSlug={orgSlug}
+          title={title}
+          dashboardTitle={dashboardTitle}
+          goBackLocation={goBackLocation}
+          onChangeTitle={newTitle => this.handleFieldChange('title', newTitle)}
```

Comment:
`Add Widget`, `Update` is still not available for metric widgets right? It's grayed out for me

Reply:
yes, we decided not to push the implementation for now